pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -324,17 +327,20 @@ func (asw *actualStateOfWorld) SetVolumeMountedByNode(
 	if err != nil {
 		return fmt.Errorf("Failed to SetVolumeMountedByNode with error: %v", err)
 	}
+	// TODO: probably safer to switch getNodeAndVolume to return pointers?
```

Comment:
do you mean to change getNodeAndVolume to return pointer for volume obj? Why it is safer?